src/strings.ts

```diff
@@ -117,6 +53,10 @@ export const connectionBlock: Dict = {
   notice: {
     info: () =>
       "You will be directed to your Management Cluster's login page through your web browser where you should enter your SSO credentials",
+    urlAlreadyUsed: () => 'This Management Cluster is already being synced',
+    nameIsEmpty: () =>
+      'Management Cluster name cannot be empty or contain whitespaces',
```

Comment:
```suggestion
      'Management Cluster name cannot be empty or contain whitespace',
```

("whitespace" without the "s" works for both singular and plural cases)

Reply:
Thank you. I don't know many thing native-speakers know. So, very appreciated.

Reply:
My pleasure!